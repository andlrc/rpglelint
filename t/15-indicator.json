[
   {
      "column":3,
      "file":"t/15-indicator.rpgle",
      "line":8,
      "message":"indicator '*INLR' is not allowed",
      "severity":"warning"
   }
]
