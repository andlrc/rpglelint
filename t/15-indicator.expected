t/15-indicator.rpgle:8:3: warning: indicator '*INLR' is not allowed [-Windicator]
   *INLR = *ON;
   ^
